client/src/components/job-applications-view-component/job-application-table-component/index.jsx

```diff
@@ -0,0 +1,163 @@
+import {
+  TablePagination,
+  TableFooter,
+  Link,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Paper,
+  TableContainer,
+} from "@mui/material";
+import DownloadIcon from "@mui/icons-material/Download";
+import PropTypes from "prop-types";
+import ApplicationStatusChipComponent from "../application-status-chips";
+
+// Future TODOs ------
+// 1. When single job specific pages and single application pages are implemented,
+//    replace `/job-posts/${row.job_posting_id}` and `/job-applications/${row.id}` to
+//    match the correct URLs
+// 2. When s3 resume download feature is complete, add download link to corresponding column
+
+function JobApplicationsTable({
+  jobApplications,
+  totalJobApplicationsNumber,
+  page,
+  rowsPerPage,
+  handlePageRowChange,
+}) {
+  const formatPhoneNumber = (phoneNumber) =>
+    phoneNumber.replace(/(\d{3})(\d{3})(\d{4})/, "$1-$2-$3");
+
+  const formatPostalCode = (postalCode) =>
+    postalCode
+      .toUpperCase()
+      .replace(/^([A-Za-z]\d[A-Za-z])(\d[A-Za-z]\d)$/, "$1 $2");
+
+  return (
+    <TableContainer
+      sx={{
+        marginTop: "35px",
+        width: "100%",
+        padding: "10px 2%",
+        boxSizing: "border-box",
+      }}
+      component={Paper}
+    >
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell sx={{ fontWeight: 700, width: "8%" }} align="left">
+              Job ID #
+            </TableCell>
+            <TableCell sx={{ fontWeight: 700, width: "15%" }} align="left">
```

Comment:
For the email, job title, and name fields, can you please add this CSS styling? The columns are quite tight together so we don't want info getting cut off! https://github.com/mui/material-ui/issues/2076#issuecomment-643899573